scripts/spec/cli_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+require 'cli'
+
+describe Build::Cli do
+  context "#parse" do
+    it "only (default)" do
+      expect(described_class.new.parse(%w()).options[:only]).to match_array %w(vsphere ovirt openstack)
+    end
+
+    it "only vsphere and ovirt" do
+      expect(described_class.new.parse(%w(-o vsphere ovirt)).options[:only]).to match_array %w(vsphere ovirt)
```

Comment:
neat / odd / scratching my head. I didn't expect that input to map to that output value.

How does it know this is not `options = {"-o" => "vsphere"} ; ARGV=["ovirt"]`

I thought it would be `-o vsphere,ovirt` or `-o vsphere -o ovirt`